File: app/src/main/java/com/arash/altafi/pdf/MainActivity.java
package com.arash.altafi.pdf;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import com.arash.altafi.pdf.pdf1.Pdf1Activity;
import com.arash.altafi.pdf.pdf2.Pdf2Activity;
import com.google.android.material.button.MaterialButton;

public class MainActivity extends AppCompatActivity {

    private MaterialButton btnPdf1;
    private MaterialButton btnPdf2;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        init();
    }

    private void init() {
        bindViews();

        btnPdf1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this , Pdf1Activity.class));
            }
        });

        btnPdf2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this , Pdf2Activity.class));
            }
        });

    }

    private void bindViews() {
        btnPdf1 = findViewById(R.id.btnPdf1);
        btnPdf2 = findViewById(R.id.btnPdf2);
    }
}

File: app/src/main/java/com/arash/altafi/pdf/MainActivity.kt
package com.arash.altafi.pdf

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.arash.altafi.pdf.databinding.ActivityMainBinding
import com.arash.altafi.pdf.pdf1.Pdf1Activity
import com.arash.altafi.pdf.pdf2.Pdf2Activity
import com.arash.altafi.pdf.pdf3.Pdf3Activity
import com.arash.altafi.pdf.pdf4.Pdf4Activity
import com.arash.altafi.pdf.pdf5.Pdf5Activity

class MainActivity : AppCompatActivity() {

    private val binding by lazy {
        ActivityMainBinding.inflate(layoutInflater)
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)

        init()
    }

    private fun init() = binding.apply {
        btnPdf1.setOnClickListener {
            startActivity(Intent(this@MainActivity, Pdf1Activity::class.java))
        }

        btnPdf2.setOnClickListener {
            startActivity(Intent(this@MainActivity, Pdf2Activity::class.java))
        }

        btnPdf3.setOnClickListener {
            startActivity(Intent(this@MainActivity, Pdf3Activity::class.java))
        }

        btnPdf4.setOnClickListener {
            startActivity(Intent(this@MainActivity, Pdf4Activity::class.java))
        }

        btnPdf5.setOnClickListener {
            startActivity(Intent(this@MainActivity, Pdf5Activity::class.java))
        }
    }

}